### PR TITLE
Update homebrew tap version after releases are published

### PR DESCRIPTION
**Pull request by jackkleeman**
We could also use this file to do doc updates etc

### CODE REVIEW

File: .github/workflows/postrelease.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+on:
+  release:
+    types: [released] # don't run on draft releases or pre-releases
+
+jobs:
+  update-tap:
+    permissions:
+      contents: write
+    secrets: inherit
+    uses: restatedev/homebrew-tap/.github/workflows/update.yml@main
```
Comment (tillrohrmann) [inline .github/workflows/postrelease.yml:12]: Are you sure that this works and we are not trying to run the `update.yml` workflow in the context of the `restate` repo?

**Review by tillrohrmann — COMMENTED** [review]
Thanks for creating this PR @jackkleeman. I had a question about where the `update.yml` workflow runs. I think it would not push a new homebrew update since it would run in the `restate` repo.

File: .github/workflows/postrelease.yml (hunk 1 above)
Comment (jackkleeman) [reply .github/workflows/postrelease.yml:12]: it will run it in this repo, so update.yml needs to checkout the right repo - ive sorted that

**Comment by jackkleeman** [comment]
but it does require that pat i requested @tillrohrmann

File: .github/workflows/postrelease.yml (hunk 1 above)
Comment (tillrohrmann) [reply .github/workflows/postrelease.yml:12]: Great, thanks.

**Review by tillrohrmann — APPROVED** [review]
The changes look good to me. I've approved the PAT you've requested. Maybe try it out again to see that it works before merging.

**Pull request merged**